src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Filter } from '../../types/Filter';
+
+interface Props {
+  activeTodos: number,
+  filter: string,
+  completedTodos: number,
+  setFilter: (filter: Filter) => void,
+}
+
+export const Footer: FC<Props> = ({
+  activeTodos,
+  filter,
+  completedTodos,
+  setFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer">
```

Comment:
Add a condition to hide the footer when there are no todos.

Reply:
I already have this in App component, add condition in Footer component also?